### ISSUE TITLE
Forks: future work

### ISSUE THREAD

**Issue by malomarrec** [sourcegraph/sourcegraph#30285]
This tracks potential follow-up work for the fork MVP #17879.

Future work:
- Fork-settings are linked to credentials. Each PAT or service account token has a setting to use forks or not. Site-admins can override this, and enforce that all changesets created by Sourcegraph are on forks.
- Should there be a "default to fork if permission error". As a user, provided there is no site-admin level fork-setting, I can set up Batch Changes to create a fork instead of a branch if creating a changeset errors due to permission issues. Blocked by: Sync write-access permission #24307

**Comment by malomarrec** [comment]
A much simpler version would be a "userForks" flag in the spec
